src/views/Phrases.vue

```diff
@@ -47,31 +47,41 @@ export default {
           this.$forceUpdate();
```

Comment:
Is it possible to rely on Vue's natural update cycle instead of thrashing the DOM?

Reply:
I'm not too sure how to resolve this issue, I've been trying to do some research, but nothing so far